src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/prisonperson/PrisonPersonMigrationMapping.kt

```diff
@@ -2,21 +2,21 @@ package uk.gov.justice.digital.hmpps.nomisvisitsmappingservice.prisonperson
 
 import org.springframework.beans.factory.annotation.Value
 import org.springframework.data.annotation.CreatedDate
-import org.springframework.data.annotation.Id
 import org.springframework.data.annotation.Transient
 import org.springframework.data.domain.Persistable
 import java.time.LocalDateTime
 
 data class PrisonPersonMigrationMapping(
 
-  @Id
   val nomisPrisonerNumber: String,
 
-  @CreatedDate
-  val whenCreated: LocalDateTime? = null,
+  val migrationType: PrisonPersonMigrationType,
 
   val label: String,
 
```

Comment:
For prison person we'll have several migrations and will use this mapping service to monitor the progress of them. Migration type will indicate what type of migration this is (currently just PHYSICAL_ATTRIBUTES). I'm expecting another column adding to save an DPS keys generated by the migration.

Note that we don't need any mappings for the sync because we have a shared business key with DPS - prisoner number.